package.json

```diff
@@ -7,6 +7,7 @@
     "url": "https://github.com/square/find-yarn-workspace-root.git"
   },
   "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
You'll also need to add this file to `files` in order to make sure it's included in the package. You can verify that you've done it correctly by doing `npm pack && tar tvfz *.tgz`.

Reply:
I've added it and it's in the package now. Sorry about that.